asf_floorplan_interpreter/pipeline/evaluation/evaluate.py

```diff
@@ -59,33 +59,57 @@ def load_process_eval_data(eval_data_file, all_training_floorplans):
         axis=1,
     )
 
+    # Combine both the door categories in the labelled data
+    def combine_door_types(double_door, door):
+        if pd.notnull(double_door):
+            if pd.notnull(door):
+                return double_door + door
```

Comment:
We should note for future labelling that by using this data, we're deciding that a double door counts for one door (otherwise we'll be feeding conflicting data). I think it's helfpul to have a reason to lean one way on this! We prevaricated a bit